ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.11.yaml

```diff
@@ -153,6 +153,13 @@ tests:
     - chain: upi-vsphere-windows-pre
     test:
     - ref: windows-unit-wicd
+- as: community-fast-forward
```

Comment:
I suggest doing this just for master ensure it works before adding to the other branches.

Reply:
Please update the PR description as well